pipeline/scripts/4/3/tier-1/test-4.2-to-latest-4.x-upgrade.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning Red Hat Ceph image based upgrade testing."
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="4.3"
+test_suite="suites/nautilus/rgw/rgw_test-update-4.2-to-4.x-latest.yaml"
```

Comment:
Sorry, this should have been caught earlier. Kindly change the suite name to `<tier>_rgw_test-4x-upgrade-to-latest.yaml`.

`update` could be changed to `upgrade` to convey the right intent of the test.

Coming to the script name... I believe it is sufficient to leave the name as `test-rgw-single-site-upgrade.sh`

Reply:
Done